### ISSUE TITLE
dark pulse doesn't break detective camera bugs

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#36748]
(WEB REPORT BY: mastercasual REMOTE: 172.93.101.77:7777)
# Revision
3dd52c94e508310598281ce3936f9f0e92325be5

# Description
dark pulse doesn't break detective camera bugs
# Steps to Reproduce
dark pulse a camera bug
# What you Expected
it to blow up.
# What Actually Happened
nothing

**Comment by hacker-on-steroids** [comment]
what the fuck is a dark pulse

**Comment by DeityLink** [comment]
Cult EMP

**Comment by hacker-on-steroids** [comment]
pretty sure regular emps don't destroy camerabugs attached to clothing either

**Comment by DeityLink** [comment]
That's correct. I think it probably should in both case though, for consistency.

**Comment by hacker-on-steroids** [comment]
seems like a trivial fix, just make clothing emp its contents as well, not just accessories
https://github.com/vgstation-coders/vgstation13/blob/3dd52c94e508310598281ce3936f9f0e92325be5/code/modules/clothing/clothing.dm#L164-L167
would stop fannypacks/webbings/labcoat pockets being emp-proof containers, too

**Comment by DeityLink** [comment]
removing my assignment since it's more of a global EMP balance feature request than a specifically cult request.
